src/app/exams/exams-view.component.ts

```diff
@@ -104,6 +106,7 @@ export class ExamsViewComponent implements OnInit, OnDestroy {
     obs.subscribe(({ nextQuestion }) => {
       if (correctAnswer === false) {
         this.incorrectAnswer = true;
```

Comment:
`incorrectAnswer` has been replaced by the new status variable, so you can remove it.  Note that you will also have to reset the value of the status variable to an empty string in line 86 of this file.